### DISCUSSION TITLE
wine: Unimplemented function KERNEL32.dll.GetSystemCpuSetInformation

### DISCUSSION THREAD

**Discussion by pdc4444** [PR3SIDENT/enshrouded-server#21] [Q&A]
Is this error message game breaking? Just now trying to get my server up and since the client doesn't appear to have a direct connect option I am not sure if  my inability to find my server in the browser is due to the server just not working correctly.

$ sudo systemctl status enshrouded.service 
● enshrouded.service - Enshrouded server - By Chevalier Pinard
     Loaded: loaded (/etc/systemd/system/enshrouded.service; enabled; vendor preset: enabled)
     Active: active (running) since Wed 2024-01-24 14:46:19 EST; 18s ago
   Main PID: 1913929 (StartEnshrouded)
      Tasks: 64 (limit: 77010)
     Memory: 115.0M
     CGroup: /system.slice/enshrouded.service
             ├─1913929 /bin/sh /home/steam/enshrouded/StartEnshroudedServer.sh
             ├─1913930 /bin/sh /usr/bin/xvfb-run --auto-servernum wine64 /home/steam/enshrouded/enshrouded_server.exe
             ├─1913953 Xvfb :99 -screen 0 1280x1024x24 -nolisten tcp -auth /tmp/xvfb-run.k2DVtJ/Xauthority
             ├─1913992 /home/steam/enshrouded/enshrouded_server.exe
             ├─1913994 /usr/lib/wine/wineserver64 -p0
             ├─1914000 C:\windows\system32\services.exe
             ├─1914003 C:\windows\system32\plugplay.exe
             ├─1914010 C:\windows\system32\winedevice.exe
             ├─1914016 C:\windows\system32\explorer.exe /desktop
             ├─1914019 C:\windows\system32\winedevice.exe
             └─1914028 winedbg --auto 8 164

Jan 24 14:46:19 ElephantBox systemd[1]: Started Enshrouded server - By Chevalier Pinard.
**Jan 24 14:46:24 ElephantBox StartEnshroudedServer.sh[1913992]: wine: Call from 0x7bc7c9e1 to unimplemented function KERNEL32.dll.GetSystemCpuSetInformation, aborting
Jan 24 14:46:24 ElephantBox StartEnshroudedServer.sh[1913992]: wine: Unimplemented function KERNEL32.dll.GetSystemCpuSetInformation called at address 000000007BC7C9E1 (thread 0009), starting debugger...**

**Comment by Bumm2000** [comment]
Hi, i had to update Debian 11 to 12, after update it worked
